modules/apps/web-experience/portlet-configuration/portlet-configuration-web/src/main/resources/content/Language.properties

```diff
@@ -6,7 +6,7 @@ any-website=Any Website
 canvas-page-url=Canvas Page URL
 copy-the-callback-url-and-specify-it-in-facebook=Copy the callback URL and specify it in Facebook.
 get-the-api-key-and-canvas-page-url-from-facebook=Get the API key and canvas page URL from Facebook.
-guest-permissions-inherited=According to the current configuration all users have the Guest role and automatically inherit permissions that belong to the Guest role.
+guest-permissions-inherited=Under the current configuration, all users have the Guest role and automatically inherit permissions that belong to the Guest role.
```

Comment:
I think we can just say: `Under the current configuration, all users automatically inherit permissions that belong to the Guest role.`

It's not necessary to say that users both have a role and inherit permissions from it, since that's what it means to have a role.  Also, users don't technically have the role (there's no actual assignment), so it's actually inaccurate.

@jorgeferrer, do you agree with this assessment?